### PR TITLE
updating the contributing doc

### PR DESCRIPTION
**Pull request by RinkiyaKeDad**
Signed-off-by: Arsh Sharma <arshsharma461@gmail.com>

### CODE REVIEW

**Review by bigash42 — APPROVED** [review]
reads well - LGTM

File: contributing.md [hunk 1]
```diff
@@ -1,96 +1,88 @@
...
-Then you just add a line to every git commit message:
+This can easily be done with the `-s' command-line option to append this automatically to your commit message.
 
 ```
-Signed-off-by: Cindy Lopez <cindy.lopez@okteto.com>
+git commit -s -m 'Meaningful commit message'
 ```
 
-If you set your `user.name` and `user.email` git configs, you can sign your commit automatically with `git commit -s`.
+> In order to use the `-s' flag for auto signing the commits, you'll need to set your `user.name`and`user.email` git configs
```
Comment (jLopezbarb) [inline contributing.md:69]:
```suggestion
> In order to use the `-s` flag for auto signing the commits, you'll need to set your `user.name`and`user.email` git configs
```

File: contributing.md [hunk 2]
```diff
@@ -1,96 +1,88 @@
...
-1. Public types, structs and funcs should be documented.
-1. It includes pertinent unit tests.
-1. Commits are signed-off (`git commit --signoff or -s`).
-1. It includes the whole template for issues and pull requests.
-1. It [references addressed issues](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords) in the PR description & commit messages.
-1. It has clear commit messages.
+The following steps will walk you through the process of opening your first pull request:
 
-## Sign your work
+##### Create a Clone of Your Fork
```
Comment (rberrelleza) [inline contributing.md:31]: this reads odd to me. maybe just go with 'Fork the repository'?

File: contributing.md [hunk 3]
```diff
@@ -122,19 +114,19 @@ make integration
...
 make lint
 ```
 
-The same command runs as part of CI on every PR.
+The same command also runs as part of CI on every PR.
 
 > This command requires that you have [golangci-lint](https://github.com/golangci/golangci-lint#install) available on your `$PATH`.
 
-## pre-commit
+### pre-commit
```
[the remark this answers is not in the document]
Comment (rberrelleza) [reply contributing.md:134]: we have pre-commit hooks, if the developer doesn't have those, the PR will fail.

File: contributing.md (hunk 2 above)
Comment (RinkiyaKeDad) [reply contributing.md]: Changed!

**Pull request merged**